README.md

```diff
@@ -103,3 +103,60 @@ Produce code coverage report
+`metadata.db` SQL database and a `cs.repo` directory containing an
+OSTree repository. Here are a few quick examples how to work with `ostree`:
+
+
+To see what references aka "branches" are in the repo:
+
+    ostree --repo=cs.repo/ refs
+    master
+
+
+To see the commit log (use less, ostree doesn't paginate like git):
+
+    ostree --repo=cs.repo/ log master
+    commit 816b63ab93a7a866d598c552f212c3a407648096521fde74cc9aa317730da8b1
+    Date:  2017-07-03 16:32:39 +0000
+    
+        Import of zsh-html into the repo
+    
+    commit f3d66bf3b40e269eac7ed62b6d6f7afdeb5059b17e1b8737aec8b52a03ea82c6
+    Date:  2017-07-03 16:32:38 +0000
+    
+        Import of zsh into the repo
+    
+    commit e370924c48e5b759c05d61b1c4960d96366a91e46eb7449a47c5064986029e50
+    Date:  2017-07-03 16:32:35 +0000
+    
+        Import of yum-rhn-plugin into the repo
+
+To find out the latest commit at `master`:
+
+    ostree --repo=cs.repo/ rev-parse master
+    816b63ab93a7a866d598c552f212c3a407648096521fde74cc9aa317730da8b1
+
+To see what files are inside a particular commit:
+
+    ostree --repo=cs.repo/ ls -RC 816b63ab93a7a866d598c552f212c3a407648096521fde74cc9aa317730da8b1
+    d00755 0 0      0 30bdab0c9a4156b26a923831f0e1c6ba6141c0a8e35c61f30e8febffe7f78de0 446a0ef11b7cc167f3b603e585c7eeeeb675faa412d5ec73f62988eb0b6c5488 /
+    d00755 0 0      0 eeae4a3c7450b786b3d9a6958ef2195c99c1867c2040d9153395e13f38b1ca7b 446a0ef11b7cc167f3b603e585c7eeeeb675faa412d5ec73f62988eb0b6c5488 /usr
+    d00755 0 0      0 0e1e668a21df0ef296e0a7287509b9bac8ae2a03bf6849954b3d0ef905b0cd9a 446a0ef11b7cc167f3b603e585c7eeeeb675faa412d5ec73f62988eb0b6c5488 /usr/share
+    d00755 0 0      0 d5c72afb0dc8197bba6249a8cdfcf155eed0e1d8bf3d3bd02a4e53f3d8c42b37 446a0ef11b7cc167f3b603e585c7eeeeb675faa412d5ec73f62988eb0b6c5488 /usr/share/doc
+    d00755 0 0      0 58efea079328274a2845195529729b190b31b06cc16b5f31c2dc1e2f88825563 446a0ef11b7cc167f3b603e585c7eeeeb675faa412d5ec73f62988eb0b6c5488 /usr/share/doc/zsh-html-5.0.2
+    -00644 0 0   1724 6c691ad9ce0f053823e3fe34795d911170bac0f78f9231389c2e9e2e1271f73a /usr/share/doc/zsh-html-5.0.2/Aliasing.html
+    -00644 0 0   1892 835ffd08dba662f47c7eb3164ae76aa5903363c63d3d606268c0ba928b725e6e /usr/share/doc/zsh-html-5.0.2/Alternate-Forms-For-Complex-Commands.html
+    -00644 0 0   1830 59184f18d807a6b9fc963b709ef82dba4d00f536388935aa79b74160fe90cbfe /usr/share/doc/zsh-html-5.0.2/Alternative-Completion.html
+    -00644 0 0   1734 b1bd5813c780a0862843fd3f468021c9358f5852c8a2029cb6857fb64dd2aed1 /usr/share/doc/zsh-html-5.0.2/Arguments.html
+    -00644 0 0  17635 89309b924ef3874172c73bab3c45eab56da0b847d6328dc387e0e1c5b3dfa633 /usr/share/doc/zsh-html-5.0.2/Arithmetic-Evaluation.html
+
+**NOTE:** `-R` means recursive, `-C` displays file checksums
+
+To check out aka export a commit into a filesystem tree:
+
+    sudo ostree --repo=cs.repo/ checkout 816b63ab93a7a866d598c552f212c3a407648096521fde74cc9aa317730da8b1 some.dir/
```

Comment:
I guess this is how import creates the repository. Doesn't work without being root/